src/tutorials/first_game_br.md

```diff
@@ -0,0 +1,166 @@
+
+### Change the Colors
+
+Navigate to the **Helper_Plane > ClientContext > Geo** group in the **Hierarchy**. There are three components that make up the airplane. One at a time, select a component and change the **Color Override** property inside the **Properties** window.
+
+<!--- TODO add image of Color Override property --->
+
+### Add a Decal
+
+You can also add more components to the airplane. Decals can be used to display a graphic on a surface.
+
+1. Inside the **Core Content** window, find the **Art Objects > Decals** category.
+2. Drag and drop the `Decal Military Symbols 01` decal into the **Geo** group within the **Hierarchy**.
+3. Position the decal onto plane.
+4. Rotate and Scale the decal to your liking.
+
+<!--- TODO add image of decal on plane --->
+
+!!! info
+    Decals can also be modified such as changing the color. With the decal selected, open the **Properties** window and scroll down to the **Smart** category. Some decals have a **Shape Index** property that change the graphic to different variations.
+
+<!--- TODO add image of decal properties --->
+
+### Update the Template
+
+Once all changes are completed, right click the **Helper_Plane** and select **Update Template From This**. After, delete **Helper_Plane** from the **Hierarchy**.
+
+<!--- TODO add image of updating template --->
+
+### Test the Game
+
+The airplane template should now be modified in game and is ready to test. The game requires at least two players to begin, so make sure to enable **Multiplayer Preview Mode** before pressing **Play**.
+
+![TwoPlayers](../img/MyFirstMultiplayer/TwoPlayers.png){: .center loading="lazy" }
+
+## Edit the Drop Zone
+
+The "Drop Zone" is a cylindrical area that represents where the player can drop from the plane. The plane will spawn near the edge of the cylinder at random and then fly across to the other side.
+
+<!--- TODO image of drop zone cylinder --->
+
+### Focus on the Drop Zone
+
+Inside the **Hierarchy**, navigate to the **Gameplay Settings > State Settings > Round Start Skydive > ServerContext** folder. Select the **DropArea** and press ++F++ to focus the scene on the large cylinder mesh.
+
+<!--- TODO image of hierarchy --->
+
+### Scale the Drop Zone
+
+Press ++R++ to enter scale mode. Change the X and Y scale so the Drop Zone is a different size.
+
+<!--- TODO image of larger drop zone --->
+
+## Change the Falling Player Movement
+
+The player has two different falling phases. When the player drops from the plane, it enters a free fall known as the **Skydive** phase. When the player presses ++space++ while skydiving then it will transition to **Parachute** phase.
+
+### Change Skydive Settings
+
+Navigate to **Gameplay Settings > State Settings > Round Start Skydive > ServerContext**. Select the **SkydivePlayerSettings** and open the Properties window. Increase the **Gravity Scale** property to `3.2` and notice how the player will fall down twice as fast with no parachute.
+
```

Comment:
If fall damage is not part of the framework, you could consider it here. It's a Game Component, I believe. A volume or template that you can drag and drop.

It's evil, but it shows the possibilities of building something pretty different.

Reply:
Looks like the framework will prevent the player from falling at a certain height and auto enable the parachute. I'll see if it's simple to disable that though.